target/linux/ramips/dts/mt7620a_iptime.dtsi

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	aliases {
+		label-mac-device = &ethernet;
+	};
+};
+
+&gpio0 {
+	status = "okay";
+};
```

Comment:
gpio0 should be enabled by default, so this node should be obsolete.

Reply:
You're right. Removed.